HeliPort/Appearance/vi.lproj/Localizable.strings

```diff
@@ -0,0 +1,65 @@
+"About HeliPort" = "Thông tin HeliPort";
+"Address: " = "Địa chỉ: ";
+"Cancel" = "Huỷ bỏ";
+"Check for Updates..." = "Kiểm tra cập nhật...";
+"Create Diagnostics Report..." = "Tạo báo cáo lỗi...";
+"Create Network..." = "Tạo mạng mới...";
+"Disconnect from: " = "Huỷ kết nối: ";
+"Dynamic WEP" = "WEP cố định";
+"Enable Wi-Fi Logging" = "Bật ghi lỗi Wifi";
+"Enter the name and security type of the network you want to join." = "Nhập tên và loại bảo mật của mạng mà bạn muốn kết nối.";
+"FUNCTION NOT IMPLEMENTED" = "CHỨC NĂNG CHƯA HOẠT ĐỘNG";
+"Find and join a Wi-Fi network." = "Tìm và kết nối một mạng Wi-fi.";
+"Interface Name: " = "Tên mạng: ";
+"Join Other Network..." = "Kết nối mạng khác...";
+"Join" = "Kết nối";
+"Network Name:" = "Tên mạng:";
+"No Network Avaliable" = "Không tìm thấy mạng";
+"No Status Information Avaliable" = "Không có thông tin trạng thái";
+"None" = "Không";
+"Open Network Preferences..." = "Mở Cài đặt mạng...";
+"Open Wireless Diagnostics..." = "Sửa lỗi Wifi...";
+"Password:" = "Mật khẩu: ";
+"Password" = "Mật khẩu";
+"Quit HeliPort" = "Thoát";
+"Reachable" = "Có thể truy cập";
+"Remember this network" = "Ghi nhớ mạng này";
+"Security:" = "Bảo mật: ";
+"Show password" = "Hiển thị mật khẩu";
+"Turn Wi-Fi Off" = "Bật Wi-Fi";
+"Turn Wi-Fi On" = "Tắt Wi-Fi";
+"Unavailable" = "Không có sẵn";
+"Unknown" = "Không rõ";
+"Unreachable" = "Không thể truy cập";
+"Username:" = "Tên người dùng:";
+"Version: " = "Phiên bản:";
+"WEP" = "WEP";
+"WPA/WPA2 Enterprise" = "WPA/WPA2 Doanh nghiệp";
+"WPA/WPA2 Personal" = "WPA/WPA2 Cá nhân";
+"WPA2 Enterprise" = "WPA2 Doanh nghiệp";
+"WPA2 Personal" = "WPA2 Cá nhân";
+"WPA2/WPA3 Enterprise" = "WPA2/WPA3 Doanh nghiệp";
+"WPA2/WPA3 Personal" = "WPA2/WPA3 Cá nhân";
+"WPA3 Enterprise" = "WPA3 Doanh nghiệp";
+"WPA3 Personal" = "WPA3 Cá nhân";
+"Wi-Fi: Connected" = "Wi-Fi: Đã kết nối";
+"Wi-Fi: Connecting" = "Wi-Fi: Đang kết nối";
+"Wi-Fi: Looking for Networks..." = "Wi-Fi: Đang tìm kiếm...";
+"Wi-Fi Network \"" = "Mạng Wi-Fi \"";
+"Wi-Fi: Off" = "Wi-Fi: Tắt";
+"Wi-Fi: On" = "Wi-Fi: Bật";
+"Wi-Fi: Status unavailable" = "Wi-Fi: Không có thông tin";
+"\" Requires Password" = "\" Yêu cầu mật khẩu";
+"    IP Address: "      = "    Địa chỉ IP: ";
+"    Router: "          = "    Router: ";
+"    Internet: "        = "    Internet: ";
+"    Security: "        = "    Bảo mật: ";
+"    BSSID: "           = "    BSSID: ";
+"    Channel: "         = "    Kênh: ";
+"    Country Code: "    = "    Mã quốc gia: ";
+"    RSSI: "            = "    RSSI: ";
+"    Noise: "           = "    Noise: ";
+"    Tx Rate: "         = "    Tỷ lệ Tx: ";
+"    PHY Mode: "        = "    Dạng PHY: ";
+"    MCS Index: "       = "    Chỉ số MCS: ";
+"    NSS: "             = "    NSS: ";             
```

Comment:
Please remove extra white spaces

Reply:
Okie. I rarely use Gihub so I dont know how to update this file?